### PR TITLE
V1 extend to merge with Master

### PR DESCRIPTION
**Pull request by NicholasDanks**
As per our discussion this pull request is to bring Master up to date.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .Rproj.user
 .Rhistory
 .RData
+pls-pm.Rproj
```
Comment (soumyaray) [inline .gitignore]: Could you make this: `*.Rproj`

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+environmentpls.RData
```
Comment (soumyaray) [inline .gitignore]: Could you make this: `*.RData`

File: AUTHORS.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Authors  
+
+### Organization  
+National Tsing Hua University Institute of Service Science   
+<http://www.iss.nthu.edu.tw>  
+
+### Original Authors
```
Comment (soumyaray) [inline AUTHORS.md:7]: Could you make this: `### Authors`

**Comment by soumyaray** [comment]
@NicholasDanks: could you look at the above line comments?  I'll merge once those are fixed.

**Comment by NicholasDanks** [comment]
Hi Prof. Changes made, you can go ahead and merge or I can do it. Thanks

**Pull request merged**